spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper.rb'
+
+describe QueueItem do
+  it { should belong_to :video }
+  it { should belong_to :user }
+  it { should validate_presence_of :video }
+  it { should validate_presence_of :user }
+
+  let(:user) { Fabricate(:user) }
+
+  describe '#video_title' do
```

Comment:
@tomtomecek you're right, I was able to use delegate_method shoulda matchers for those 3 methods, thanks!